include/sys/dkioc_free_util.h

```diff
@@ -48,7 +48,7 @@ typedef struct dkioc_free_list_s {
 } dkioc_free_list_t;
 
 static inline void dfl_free(dkioc_free_list_t *dfl) {
-	kmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
+	vmem_free(dfl, DFL_SZ(dfl->dfl_num_exts));
```

Comment:
It would make perfect sense.  I think the only reason we don't have one is because illumos doesn't have one.  I do like the idea of adding one.  We've already got the kmem/vmem difference from the upstream and I suppose a ```dfl_alloc()``` would make the intention more obvious for future porting efforts.